lib/formdata/0.0.1/index.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * FormData polyfill for k6
+ * Copyright (C) 2021 Load Impact
+ * License: MIT
+ *
+ * This simplifies the creation of multipart/form-data requests from k6 scripts.
+ * It was adapted from the original version by Rob Wu[1] to remove references of
+ * XMLHttpRequest and File related code which isn't supported in k6.
+ *
+ * [1]: https://gist.github.com/Rob--W/8b5adedd84c0d36aba64
+ **/
+(function(exports) {
+  if (exports.FormData) {
+    // Don't replace FormData if it already exists
+    return;
+  }
+  // Export variable to the global scope
+  exports.FormData = FormData;
+
+  function FormData() {
+    // Force a Constructor
+    if (!(this instanceof FormData)) return new FormData();
+    // Generate a random boundary - This must be unique with respect to the
+    // form's contents.
+    this.boundary = '------RWWorkerFormDataBoundary' + Math.random().toString(36);
+    this.parts = [];
+    /**
+    * Internal method.
+    * @param inp String | ArrayBuffer | Uint8Array  Input
+    */
+    this.__toByteArray = function(inp) {
+      var arr = [];
+      var i = 0, len;
+      if (typeof inp == 'string') {
+        for (len = inp.length; i < len; ++i)
```

Comment:
This should probably be `i++` or `i+=1`.

Reply:
I left this as it was in the original gist, but I also always use the postfix notation (`i++`).

In this case it shouldn't matter, right? Is there a particular reason to not use `++i`?

Reply:
No you're right it definitely works as is.